kola/tests/misc/network.go

```diff
@@ -127,6 +127,7 @@ func NetworkListeners(c cluster.TestCluster) {
 		{"tcp", "22", "systemd"},          // ssh
 		{"udp", "68", "systemd-network"},  // dhcp6-client
 		{"udp", "546", "systemd-network"}, // bootpc
+		{"tcp", "10010", "containerd"},    // containerd
```

Comment:
will this make the test fail on non-edge builds?

Reply:
Why would it make the non-edge tests fail? On the contrary, it will actually fix failures on both edge and non-edge channels.

The change in the cgroup driver of Docker should not change anything on non-edge builds, because non-edge builds will always have `--cgroup-driver=cgroupfs`. The cgroup driver change will only affect edge builds, where `--cgroup-driver=systemd` is being used. However, I wanted to make kola tests detect both cgroupfs and systemd, regardless of channels. That's because in the future we could have situations where the systemd cgroup driver is enabled in non-edge channels. Doing that, we can also avoid applying the edge changes back again to flatcar-master.

The change in NetworkListeners would also not affect existing test cases, because it's just adding another entry to the whitelist, AFAIU.

Reply:
I thought non-edge releases don't have `containerd` and that whitelist was strict (if the process name is in the whitelist and not on the system, it would fail). Just wanted to check but if it's just a whitelist then I'm fine with it.